### DISCUSSION TITLE
Humidity

### DISCUSSION THREAD

**Discussion by thejob78** [jmcollin78/versatile_thermostat#491] [Ideas]
Hello, it would be very useful if the climate integration and the UI Card could manage/visualize the humidity of a specific room. This is particularly useful for those who have underfloor heating and want to use it for cooling as well. It helps in calculating the dew point in the room.

w point in the room.

**Comment by jmcollin78** [comment]
Already in an issue: https://github.com/jmcollin78/versatile_thermostat/issues/167
